codes/typescript/chapter_heap/my_heap.ts

```diff
@@ -114,6 +114,11 @@ class MaxHeap {
     public print(): void {
         printHeap(this.maxHeap);
     }
+
+    /* 对堆取相反数 */
+    public invertedNums(): number[] {
```

Comment:
It'd be better to replace this functions with `getMaxHeap()`. And invert the numbers out of the class.